packages/design-system/src/components/Menu/Menu.tsx

```diff
@@ -30,17 +32,18 @@ const Menu = React.forwardRef<React.ReactElement, MenuProps>(
 			<S.Nav
 				isCollapsed={isCollapsed}
 				role="navigation"
-				aria-label="main navigation"
+				aria-label={t('MENU_MAIN_NAVIGATION', 'main')}
```

Comment:
There is no need to add navigation in the label as any screenreader will announce it as it's in a nav element.